api/api.go

```diff
@@ -26,11 +27,14 @@ import (
 	"github.com/imdario/mergo"
 )
 
+//go:embed dev-console-app/*
+var devConsoleApp embed.FS
+
 func New(config *core.Config, apiRoot string) *ExtensionsApi {
 	mux := mux.NewRouter()
 
 	mux.HandleFunc("/", func(rw http.ResponseWriter, r *http.Request) {
-		http.Redirect(rw, r, apiRoot, http.StatusTemporaryRedirect)
+		http.Redirect(rw, r, "/dev-console-app", http.StatusPermanentRedirect)
```

Comment:
Do we need the `-app` suffix here? Increases the size of the url without much gain. Feels unnecessary.